### ISSUE TITLE
nkn version 1.2.1 ERR_PACKAGE_PATH_NOT_EXPORTED when running example

### ISSUE THREAD

**Issue by sce9sc** [nknorg/nkn-sdk-js#82]
After installing the version 1.2.1 and trying to run an example I get

```node:internal/modules/cjs/loader:488
      throw e;
      ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in /Users/sce9sc/Documents/Work/testNknLatestNode/node_modules/@nkn/ncp/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:429:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:683:3)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/sce9sc/Documents/Work/testNknLatestNode/node_modules/nkn-sdk/lib/multiclient/multiclient.js:8:35) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

**Comment by yilunzhang** [comment]
I've got this issue before in another project. It seems to be a npm issue. The solution that worked for me was to remove `node_moodules` and `package.lock`/`yarn.lock` (depending on which on you use) and do `npm install`/`yarn` again.

**Comment by sce9sc** [comment]
I tried it in a new and clean project with no module and no package.json and still got the same message.

**Comment by yilunzhang** [comment]
I tried a few nodejs version and confirmed this issue with some new nodejs versions. This should have been fixed in v1.2.2 now.
